src/Appwrite/Platform/Workers/Databases.php

```diff
@@ -292,23 +293,29 @@ private function deleteAttribute(Document $database, Document $collection, Docum
 
                 if ($e instanceof DatabaseException) {
                     $attribute->setAttribute('error', $e->getMessage());
-                    if (!$relatedAttribute->isEmpty()) {
+                    if (! $relatedAttribute->isEmpty()) {
                         $relatedAttribute->setAttribute('error', $e->getMessage());
                     }
                 }
+
                 $dbForProject->updateDocument(
                     'attributes',
                     $attribute->getId(),
                     $attribute->setAttribute('status', 'stuck')
                 );
-                if (!$relatedAttribute->isEmpty()) {
+
+                if (! $relatedAttribute->isEmpty()) {
                     $dbForProject->updateDocument(
                         'attributes',
                         $relatedAttribute->getId(),
                         $relatedAttribute->setAttribute('status', 'stuck')
                     );
                 }
 
+                if ($e instanceof Dependency) {
+                    return;
+                }
+
```

Comment:
Why do we want to skip throwing here?

Reply:
We do not need this case to get to Sentry since it is handled.